git-cliff-core/src/release.rs

```diff
@@ -96,6 +137,16 @@ impl Release<'_> {
 		self.calculate_next_version_with_config(&Bump::default())
 	}
 
+	/// Returns a copy of the release with aggregated statistics populated.
```

Comment:
Copy sounds like the `Copy` trait in Rust. Can you rephrase this comment?

Reply:
Good point, I'll change the wording to avoid confusion with the `Copy` trait.